.claude/skills/bob-hunt/SKILL.md

```
@@ -76,28 +73,28 @@ MCP-owned session artifacts:
 - If `state.pending_wave` is non-null, call `bounty_apply_wave_merge({ target_domain, wave_number: state.pending_wave, force_merge, force_merge_reason })` and use `result.data`. When `force_merge` is true, `force_merge_reason` must explain the missing/invalid handoffs and why reconciliation is safe.
 - If status is `"pending"`, report `Wave N pending: X/Y handoffs received. Resume again later, or run /bob-hunt resume [domain] force-merge to reconcile now.` Then stop.
 - If status is `"merged"`, continue with returned `state`, `readiness`, `merge`, and `findings`.
 - Pending-wave reconciliation happens only on explicit re-entry or after all background hunters complete, never in the same turn that launched hunters.
 
 ## PHASE 1: RECON
-Call `bounty_init_session({ target_domain, target_url })`.
+Call `bounty_init_session({ target_domain, target_url, deep_mode })`.
 
-Spawn exactly one recon agent and wait:
+Spawn exactly one recon agent by resolved `deep_mode`, then wait:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)